tests/api_connexion/endpoints/test_log_endpoint.py

```diff
@@ -113,6 +113,7 @@ def _prepare_db(self):
         dagbag = self.app.dag_bag  # pylint: disable=no-member
         dag = DAG(self.DAG_ID, start_date=timezone.parse(self.default_time))
         dag.sync_to_db()
+        dagbag.dags.pop(self.DAG_ID, None)
```

Comment:
These `dict.pop()` calls resets the `app.bag_dag` state across tests so they can be re-populated for new things to test. An alternative fix is to use a different DAG ID for each test, which generates a larger diff, and probably slightly slower (DAGs created here need to be sync-ed into the db, so more DAG IDs = more db inserts; I didn’t actually benchmark this though).